extensions/amp-story/1.0/amp-story-reaction-poll-binary.js

```diff
@@ -0,0 +1,177 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {AmpStoryReaction, ReactionType} from './amp-story-reaction';
+import {CSS} from '../../../build/amp-story-reaction-poll-binary-1.0.css';
+import {createShadowRootWithStyle} from './utils';
+import {dev, devAssert} from '../../../src/log';
+import {htmlFor} from '../../../src/static-template';
+import {scopedQuerySelector, scopedQuerySelectorAll} from '../../../src/dom';
+import {toArray} from '../../../src/types';
+
+/** @const {string} */
+const TAG = 'amp-story-reaction-poll-binary';
+
+/** @const @enum {number} */
+export const FontSize = {
+  EMOJI: 28,
+  SINGLE_LINE: 16,
+  DOUBLE_LINE: 14,
+};
```

Comment:
According to mocks, the font size on post-select depends on the content of the poll, so these are the font sizes.